src/index.js

```diff
@@ -1,13 +1,60 @@
-import _ from 'lodash';
 import './style.css';
 
-function component() {
-  const element = document.createElement('div');
+const refresh = document.querySelector('.btn-refresh');
+const form = document.getElementById('form');
+const submit = document.getElementById('add-btn');
+const listScore = document.getElementById('score_list');
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['', ''], ' ');
+const gameId = 'ngRRYTtqlgOlpU5S061T';
 
-  return element;
-}
+const submitScore = async (userName, userScore) => {
+  const response = await fetch(
+    `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores`,
```

Comment:
- [ ] I really have tried all the ways in order to run your application on my machine but none is working. I really do not know how you run your application. If you run it in a special way, please specify it in your README and add the steps to run it 😃 💡. I tried the usual way of starting the webpack dev server by running `npm start` and it was not working. At first, it showed an error that the webpack dev server is not installed. I installed it. I run it again and it kept on showing the same error. I really don't know the cause up to now. Please try to fix it. However, if you run your application in a new way and it is working, please provide the information in both README and the Pull Request description. Here is the screenshot showing error 👇 
![image](https://user-images.githubusercontent.com/90222110/180461109-53b86983-7af2-4eac-9eaa-ac01d59c0a98.png)